qa/tests/src/org/jboss/jbossts/qa/Utils/JDBCProfileStore.java

```diff
@@ -31,126 +31,108 @@
-	}
+        return Integer.parseInt((String) _profile.get(profileName + "_NumberOfDrivers"));
+    }
 
-	public static String driver(String profileName, int driverNumber)
-			throws Exception
-	{
-		loadProfile();
+    public static String driver(String profileName, int driverNumber) throws Exception {
+        loadProfile();
 
-		return (String) _profile.get(profileName + "_Driver" + driverNumber);
-	}
+        return (String) _profile.get(profileName + "_Driver" + driverNumber);
+    }
 
-	public static String databaseURL(String profileName)
-			throws Exception
-	{
-		loadProfile();
+    public static String databaseURL(String profileName) throws Exception {
+        loadProfile();
 
-		return (String) _profile.get(profileName + "_DatabaseURL");
-	}
+        return (String) _profile.get(profileName + "_DatabaseURL");
+    }
 
-	public static String databaseUser(String profileName)
-			throws Exception
-	{
-		loadProfile();
+    public static String databaseUser(String profileName) throws Exception {
+        loadProfile();
 
-		return (String) _profile.get(profileName + "_DatabaseUser");
-	}
+        return (String) _profile.get(profileName + "_DatabaseUser");
+    }
 
-	public static String databasePassword(String profileName)
-			throws Exception
-	{
-		loadProfile();
+    public static String databasePassword(String profileName) throws Exception {
+        loadProfile();
 
-		return (String) _profile.get(profileName + "_DatabasePassword");
-	}
+        return (String) _profile.get(profileName + "_DatabasePassword");
+    }
 
-	public static String databaseDynamicClass(String profileName)
-			throws Exception
-	{
-		loadProfile();
+    public static String databaseDynamicClass(String profileName) throws Exception {
+        loadProfile();
 
-		return (String) _profile.get(profileName + "_DatabaseDynamicClass");
-	}
+        return (String) _profile.get(profileName + "_DatabaseDynamicClass");
+    }
 
-	// new methods for jndi
-	public static String binding(String profileName)
-			throws Exception
-	{
-		loadProfile();
+    // new methods for jndi
+    public static String binding(String profileName) throws Exception {
+        loadProfile();
 
-		return (String) _profile.get(profileName + "_Binding");
-	}
+        return (String) _profile.get(profileName + "_Binding");
+    }
 
-	public static String databaseName(String profileName)
-			throws Exception
-	{
-		loadProfile();
+    public static String databaseName(String profileName) throws Exception {
+        loadProfile();
 
-		return (String) _profile.get(profileName + "_DatabaseName");
-	}
+        return (String) _profile.get(profileName + "_DatabaseName");
+    }
 
-	public static String host(String profileName)
-			throws Exception
-	{
-		loadProfile();
+    public static String serviceName(String profileName) throws Exception {
+        loadProfile();
 
-		return (String) _profile.get(profileName + "_Host");
-	}
+        return (String) _profile.get(profileName + "_DatabaseName");
```

Comment:
I think you should consider to remove this serviceName function (and please see below for some more context)

Reply:
Ops, copy&paste mistake. `_DatabaseName` at line 93 should have been `_ServiceName`. Also, the function I was calling was `getOrDefault`...I will update the PR. Thanks for spotting that :-)